plugins/widget/plugin.js

```diff
@@ -3219,6 +3220,11 @@
 	}
 
 	function onBlockWidgetDrag( evt ) {
+		// Allow to drag widget only with left mouse button (#711).
+		if ( typeof evt.data != 'undefined' && evt.data.$.button != 0 ) {
```

Comment:
This code doesn't work in IE8 as it uses different values for buttons (0 means "no button" and 1 means "left button"). We have at least two implementations of such logic inside editor, you could look at e.g. [`copyformatting` plugin](https://github.com/ckeditor/ckeditor-dev/blob/f7deea1b22b7a975ee02adff0546e2b338301ed4/plugins/copyformatting/plugin.js#L13-L31).